layers/+intl/japanese/config.el

```diff
@@ -0,0 +1,13 @@
+;;; config.el --- Japanese Layer configuration File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Kenji Miyazaki <kenjimyzk@gmail.com>
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; Variables
+
+(defvar migemo-dictionary "/usr/local/share/migemo/utf-8/migemo-dict")
```

Comment:
`migemo-dictionary` is defined in [migemo](https://github.com/emacs-jp/migemo/blob/master/migemo.el#L80). So you don't have to redefine it here. 

Also it's a good thing to document variables defined in `config`.